jekyll/_components/panes.md

```diff
@@ -0,0 +1,209 @@
+
+<div markdown="1">
+The arrow icon indicates that a particular pane can be expanded and collapsed and what the current expanded state is (a downward pointing arrow indicates that the pane is expanded). When present, The arrow appears as the leftmost element in the pane header.
+</div>
+</div>
+
+### Button(s)
+
+<div class="grid-2" markdown="1">
+![Pane buttons](../images/components/pane/pane-buttons.png)
+
+<div markdown="1">
+Primary functions belonging to a pane may be exposed within the pane header to increase visibility and to reduce the need for an additional toolbar. 
+
+The Watch Expressions pane in the Debugger is a good example, the pane header offers an “Add Watch Expression” button (represented by a plus symbol) allowing users to quickly add new expressions.
+Functional buttons must appear right aligned within the pane header.
+</div>
+</div>
+
+
+
+
+
+## Style
+
+<div data-tabs>
+  <input type="radio" name="tstyle" id="style-generic" checked="checked">
+  <label for="style-generic">Light</label>
+  <div data-tab markdown="1">
+
+### Color
+
+<div class="grid-2" markdown="1">
+![pairing example](../images/components/pane/pane-light-mode.png)
+
+<div markdown="1">
+#### Pane
+
+Background: white `#ffffff`
+
+Border/Splitter: Grey 25 `#e0e0e2`
+
+#### Header
+
+Background: Grey 10 `#f9f9fa`
+
+Text: Grey 90 `#0c0c0d`
+
+Arrow: `rgba(0, 0, 0, 0.3)`
+
+Icon: Grey 60 `#4a4a4f`
+
+#### Header Hover
+
+Background: Grey 20 `#ededf0`
+
+Cursor: `default`
+</div>
+</div>
+  </div>
+
+
+  <input type="radio" name="tstyle" id="style-success">
+  <label for="style-success">Dark</label>
+  <div data-tab markdown="1">
+  
+### Color
+
+<div class="grid-2" markdown="1">
+![pairing example](../images/components/pane/pane-dark-mode.png)
+
+<div markdown="1">
+#### Pane
+
+Background: Grey 85 `#1b1b1d`
+
+Border/Splitter: `#3c3c3d` *Photon color needed*
+
+#### Header
+
+Background: `#222225` *Photon color needed*
+
+Text: Grey 40 `#b1b1b3`
+
+Arrow: `rgba(255, 255, 255, 0.4)`
+
+Icon: Grey 40 `#b1b1b3`
+
+#### Header Hover
+
+Background: Grey 80 `#2a2a2e`
+
+Cursor: `default`
+</div>
+</div>
+  </div>
+</div>
+
+
+
+
+
+### Sizes
+
+<div class="grid-2" markdown="1">
+![Pane sizes and measurements](../images/components/pane/pane-sizes.svg)
+
+<div markdown="1">
+
+#### Pane
+
+Padding `4px 20px`
+
+#### Header
+
+Height: `24px`
+
+Text: `12px, regular 400`
+
+Padding: `4px 20px`
+
+**Note:** The 20px left padding is the sum of 6px padding either side of the 8px arrow icon.
+
+</div>
+</div>
+
+
+
+<div class="grid-2" markdown="1">
+![Pane icon sizes and measurements](../images/components/pane/pane-icon-sizes.svg)
+
+<div markdown="1">
+
+#### Arrow
+
+Width: `8px`
+
+Height: `8px`
+
+Containing an `8px` x `5px` SVG
+
+#### Icons
+
+width: `10px`
+
+height: `10px`
+
+</div>
+</div>
+
+
+## Behaviours
+
+### Expand and collapse
+
+<div class="grid-2" markdown="1">
+![Pane behaviour](../images/components/pane/pane-behaviour.gif)
```

Comment:
If you look at https://github.com/FirefoxUX/photon/pull/334/files#diff-80eddb616ebe9c94995bc02d10ea6519R30 you'll see that we recently added the ability to have a play/pause frame for animations, by adding the line: `{:.animated}` underneath, and providing files named `pane-behaviour-play.png` and `pane-behaviour-pause.png`. I think that might be something you want to take advantage of… 😉